api-lib/gql/__generated__/zeus/const.ts

```diff
@@ -5688,6 +5692,9 @@ export const ReturnTypes: Record<string, any> = {
     id: 'Int',
     profile: 'profiles',
   },
+  MarkClaimedOutput: {
+    ids: 'Int',
```

Comment:
curious about this being plural but being a singular int

Reply:
oh good catch, this should be an array type

Reply:
idunno what's going on here, but the zeus index.ts files have the correct type:
```
['MarkClaimedOutput']: {
  ids: Array<number>;
};
```